src/server/bg_services/log_replication_scanner.js

```diff
@@ -90,7 +90,14 @@ class LogReplicationScanner {
 
                 const sync_versions = rule.sync_versions || false;
 
-                await this.process_candidates(src_bucket, dst_bucket, candidates.items, sync_versions, rule_id, replication_id);
+                const sync_deletions = rule.sync_deletions || false;
+
+                if (!sync_versions && sync_deletions) {
+                    throw new Error('Cannot enable sync_deletions when sync_versions is not enabled');
+                }
```

Comment:
This is not true.
in log based you can have sync_deletions without sync_versions

Reply:
Removed, thanks